### PR TITLE
edge: add support for lack of end-of-candidates

### PR DESCRIPTION
**Pull request by fippo**
This adds a timeout based end-of-candidates indication. Edges native 1.0 has a similar hack but doesn't complain in the console afaics.

@KaptenJansson oh hey, this would make apprtc work... ;-)

### CODE REVIEW

File: src/js/edge/rtcpeerconnection_shim.js [hunk 1]
```diff
@@ -879,6 +879,24 @@ module.exports = function(edgeVersion) {
         });
       }
     });
+
+    // check whether addIceCandidate({}) was called within four seconds after
+    // setRemoteDescription.
+    window.setTimeout(function() {
+      if (!self && self.transceivers) {
```
Comment (KaptenJansson) [inline src/js/edge/rtcpeerconnection_shim.js]: should this be `!self.transceivers `?

Comment (fippo) [reply src/js/edge/rtcpeerconnection_shim.js]: duh. squashed

**Comment by fippo** [comment]
(this one needs a minor version bump)

**Comment by MirkoBonadei** [comment]
@fippo lgtm. Do you want me to merge and bump the minor revision?

**Comment by fippo** [comment]
merge but no bump yet please -- there is lots of other stuff ;-)

**Comment by MirkoBonadei** [comment]
Done, thanks.
I will try to have a look at the Travis build today.

**Comment by fippo** [comment]
and yes, lets keep in mind that the next version has to be a minor bump since this is a feature

**Pull request merged**